x-pack/platform/plugins/shared/cases/public/components/case_view/case_view_tabs.test.tsx

```diff
@@ -122,6 +123,20 @@ describe('CaseViewTabs', () => {
     );
   });
 
+  it('shows the events tab as active', async () => {
```

Comment:
Could you add a test that checks that the events tab is not shown when the feature is disabled?